examples/Default.example

```diff
@@ -3,41 +3,66 @@ class Example extends React.Component {
   constructor(props) {
     super(props);
     cookie('accepts-cookies', '');
-    this.state = {
```

Comment:
I removed the `dismissOnScroll` logic, as it seems to be broken (not sure why the `onScroll` listener doesn't fire).